test/integration/parcel-operators-handler.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { ParcelOperators } from '../../src/adapters/parcel-rights-fetcher'
+import { test } from '../components'
+import { generateRandomAddress } from '../helpers'
+
+test('integration tests for parcel operators handler', function ({ components, stubComponents }) {
+  it('should respond with a 200 and the correct owner and operator when parcel exists', async () => {
+    const { localFetch } = components
+    const ownerAddress = generateRandomAddress()
+    const operatorAddress = generateRandomAddress()
+    const parcelX = 10
+    const parcelY = 20
+
+    stubComponents.parcelRightsFetcher.getOperatorsOfParcel.resolves({
+      owner: ownerAddress,
+      operator: operatorAddress
+    } as ParcelOperators)
+
+    const r = await localFetch.fetch(`/parcels/${parcelX}/${parcelY}/operators`)
+
+    expect(r.status).toBe(200)
+    expect(await r.json()).toEqual({
+      owner: ownerAddress,
+      operator: operatorAddress
+    })
+  })
+
+  it('should respond with a 200 and the correct owner and operator when estate exists', async () => {
```

Comment:
As we're mocking `getOperatorsOfParcel` the test is the same as the one above, would you mind removing it?